src/locales/en/default.yml

```diff
@@ -44,7 +44,8 @@ govuk:
         - href: "https://signin.account.gov.uk/privacy-notice"
           text: "Privacy notice"
         - href: "https://home.account.gov.uk/contact-gov-uk-one-login"
-          text: "Support"
+          text: "Support (Opens in new tab)"
```

Comment:
Minor one but if we are trying to remain consistent the O in opens should be lowercase